pages/common/docker-images.md

```diff
@@ -0,0 +1,20 @@
+# docker images
+
+> Manage Docker images.
+> More information: <https://docs.docker.com/engine/reference/commandline/images/>.
+
+- List all Docker images:
+
+`docker images`
+
+- List all Docker images with intermediate included:
+
+`docker images -a`
+
+- List all Docker images numeric IDs:
+
+`docker images -q`
+
+- List all Docker images not used by any container:
```

Comment:
Is there any way of deleting these automatically with this command?

Reply:
No. For that one would use `docker rmi`.

Reply:
Right. That  would be a separate page then.

Reply:
there are many subcommands missing in **_tldr_**, but slowly we are getting there :)